src/test/java/marquez/api/JobRunIntegrationTest.java

```diff
@@ -11,14 +11,13 @@
 import javax.ws.rs.client.Entity;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
-import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class JobRunIntegrationTest extends JobRunBaseTest {
   private static Logger LOG = LoggerFactory.getLogger(JobRunIntegrationTest.class);
 
-  @Test
+  // @Test
```

Comment:
Why are the tests being disabled?
Tactically if you want to disable a set of tests, please use `@Ignore("disable reason here")` at the class level. If you want to disable only some tests, you can also use `@Ignore` at the method level.

Reply:
missed this, thanks

Reply:
ah, this test is no longer valid, which is probably why I disabled. will mark ignored -- feel free to re-enable if you use these tests for the endpoints